Zend/zend_compile.h

```diff
@@ -907,9 +907,10 @@ ZEND_API zend_string *zend_type_to_string(zend_type type);
 #define ZEND_FETCH_TYPE_MASK	0xe
 
 /* Only one of these can ever be in use */
-#define ZEND_FETCH_REF			1
-#define ZEND_FETCH_DIM_WRITE	2
-#define ZEND_FETCH_OBJ_FLAGS	3
+#define ZEND_FETCH_REF				1
+#define ZEND_FETCH_DIM_WRITE		2
+#define ZEND_FETCH_DIM_UNSET_FLAG	3
```

Comment:
If this is deliberately the same value as ZEND_FETCH_OBJ_FLAGS, document the reason why?

Reply:
Ah, this is just because the flag that was removed from the same place a while ago had the same value. I'll have another shot with this flag since I couldn't make it work for me. :)